app/konnect/servicehub/service-documentation.md

```diff
@@ -9,90 +9,27 @@ documentation to the Dev Portal. Publishing services to the Dev Portal is the on
 
 ## API specification
 
 API specifications, or specs, can be uploaded and attached to a specific version within your Dev Portal.
 You can have different API specs that correspond to different service versions.
 {{site.konnect_short_name}} accepts OpenAPI (Swagger) specs in YAML or JSON.
 
-### Upload a version spec
-
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service version, then follow these steps:
-
-1. In the **Version Spec** section, click **Upload Spec**.
-
-1. Click **Upload Spec** again to upload your OpenAPI specification file.
-
-    The spec must be in YAML or JSON format. You
-    can use [vitalsSpec.yaml](/konnect/vitalsSpec.yaml) as a sample spec to test this functionality.
-    If the service was previously published to the Dev Portal, the documentation
-    for the service gets automatically updated with your changes. If not,
-    [publish](/konnect/servicehub/service-documentation/#publishing) the service.
-
-### Update a version spec
-
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service version. Upload a new spec:
-
-* In the **Version Spec** section, click the spec, then click **Replace**.
-Choose a new spec to replace the existing one.
-
-The published documentation for the service is automatically updated in the Dev Portal.
-
-### Delete a version spec
-
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service version.
-Delete the spec:
-
-* In the **Version Spec** section, click the {% konnect_icon cogwheel %} icon, then **Delete** to remove the spec.
-
-Deleted files are permanently removed from the Dev Portal.
-
-## Publishing
-
-### Publish a service {#publish}
-
-Publish a service and its API specs to the Dev Portal. Publishing a service makes it available to developers in your organization.
-
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.
-Publish the service:
-
-* From the **Service actions** drop-down menu, select **Publish to portal**.
-
-### Unpublish a service {#unpublish}
-
-Unpublish a service to remove it from the Dev Portal. Removing a service from the Dev Portal is not permanent, and you can republish it at any time.
-
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.
-Unpublish the service:
+<p align="center">
+  <img src="/assets/images/docs/konnect/konnect_service_docs_spec_edit.png" />
+</p>
 
-* From the **Service actions** drop-down menu, select **Unpublish from portal**.
+Once you've uploaded the spec, you can also preview the way the spec will render, including the methods available, endpoint descriptions, and example values. You'll also see the publication status. 
```

Comment:
It would be nice to also call out we support the ability to Filter by tag in Full Page View.